src/components/Task.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <li :class="{done: task.taskState}">
+    <div>
+      <input
+        type="checkbox"
+        v-model="task.taskState"
+        @change="clickOnCheckbox"
+      >
+      <span class="title">
+        {{task.taskText}}
+      </span>
+    </div>
+    <button
+      class="item-btn"
+      @click="onClickDeltTaskBtn"
+    >Delete</button>
+  </li>
+</template>
+
+<script>
+export default {
+  name: 'Task',
+  props: ['task', 'listId'],
```

Comment:
In very rare cases you need to use Objects for props, so please split **task object** to few different props